### PR TITLE
1 set up myproject

### LINKED ISSUE #1: set-up-myproject
### Issue: set up my project
- For it: I will :
- Create a new directory and initialize it with git.
- Use npm init -y command to create package.json file.
- Create the entry point for your JavaScript code called index.js
- Create index.css file for your styles.
- Create the main index.html file. Link your index.js and index.css files there.
- Create a directory called modules.

### Goal
- The goal is to prepare the structure of my project

### PR DESCRIPTION
**Pull request by IvonneBenitesRodriguez**
### About this project
- This is a project about a single page app. 
- It is a page that offers the user an input to place his/her favorite book with the title book and the name of the author. 
- It also displays the current time, and it has a remove button in case the user wants to remove that input.
- It contains the following files: index.html, index.js, index.css 
- It contains a module section with the following files: app.js, time.js, display.js 
   We used the mentioned modules to build our index.js and give the dynamic functionality to our single page.
-  The usage is about to import and export classes and functions which appeared in the display.js and app.js files.
- In the index.html file we referenced the id to manipulate the data used in the index.js

### Goal
 - The goal of this project is to understand how to use ES6 modules and how to import files to a new js file.

### CODE REVIEW

File: modules/time.js
Comment (Efiamotu-1) [inline modules/time.js:1]: - [ ] You are currently not following the project requirement which requires you to import  `{Datetime}` from `luxon` and use it to display the date in your app 
lux

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @IvonneBenitesRodriguez , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/time.js
Comment (IvonneBenitesRodriguez) [reply modules/time.js:1]: Dear @Efiamotu-1 👋🏾
Very nice to meet you, thank you for your kind messages 🙏🏿
I appreciate it, I have modified what you suggested me, 
Best regards,

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand: @IvonneBenitesRodriguez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 

:tada: :champagne:  :tada: :champagne: 

## Highlights :green_circle: 

- `luxon` library has been used to display date on the app as requested :rocket: 

## Optional suggestions
**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**